### PR TITLE
Mgreenegit patch 1

### PR DESCRIPTION
**Pull request by mgreenegit**
Updates LCM pages to ensure the service in Azure and on-premises options are both explained well.

Also modifies the TOC page by putting both LCM pages at the same level, with a top level folder rather than the top level item being a standalone page.

Attention: @Indhukrishna @OaaSTeam @eslesar

### CODE REVIEW

**Comment by saldana** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-ChildItem.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/reference/3.0/Microsoft.PowerShell.Management/Get-ChildItem.md) | :x:Error | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Management%2FGet-ChildItem.md)
[reference/3.0/Microsoft.PowerShell.Utility/Get-Date.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/reference/3.0/Microsoft.PowerShell.Utility/Get-Date.md) | :x:Error | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FGet-Date.md)
[]() | :x:Error | | [Details](#user-content-)
[dsc/metaConfig.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig.md) | :warning:Warning | | [Details](#user-content-dsc%2FmetaConfig.md)
[dsc/metaConfig4.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig4.md) | :warning:Warning | | [Details](#user-content-dsc%2FmetaConfig4.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Management%2FGet-ChildItem.md"></a>
### [reference/3.0/Microsoft.PowerShell.Management/Get-ChildItem.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/reference/3.0/Microsoft.PowerShell.Management/Get-ChildItem.md)
 - **[Error]** Error transforming model &quot;W:\u4dy-o-unknown\raw-\m32stmng.v4m\rawmodel\powershell-3.0\module\Microsoft.PowerShell.Management\Get-ChildItem.raw.json&quot; generated from &quot;3.0/Microsoft.PowerShell.Management/Get-ChildItem.md&quot; using &quot;AzureCli.html.primary.js&quot;. Error running GetOptions function inside template preprocessor: 

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FGet-Date.md"></a>
### [reference/3.0/Microsoft.PowerShell.Utility/Get-Date.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/reference/3.0/Microsoft.PowerShell.Utility/Get-Date.md)
 - **[Error]** Error transforming model &quot;W:\u4dy-o-unknown\raw-\m32stmng.v4m\rawmodel\powershell-3.0\module\Microsoft.PowerShell.Utility\Get-Date.raw.json&quot; generated from &quot;3.0/Microsoft.PowerShell.Utility/Get-Date.md&quot; using &quot;AzureCli.html.primary.js&quot;. Error running GetOptions function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\u4dy-o-unknown\raw-\m32stmng.v4m\rawmodel\powershell-3.0\module\Microsoft.PowerShell.Management\Get-ChildItem.raw.json&quot; generated from &quot;3.0/Microsoft.PowerShell.Management/Get-ChildItem.md&quot; using &quot;AzureCli.html.primary.js&quot;. Error running GetOptions function inside template preprocessor: 
 - **[Warning]** &#39;W:\u4dy-s\gallery/index.md&#39; would be overwritten by redirection rule configured in master redirection file .openpublishing.redirection.json. Please remove the original file to resolve this warning.
 - **[Warning]** &#39;W:\u4dy-s\jea/index.md&#39; would be overwritten by redirection rule configured in master redirection file .openpublishing.redirection.json. Please remove the original file to resolve this warning.

<a name="dsc%2FmetaConfig.md"></a>
### [dsc/metaConfig.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig.md)
 - **[Warning]** Ignore metadata: ms.locale. They are generated by open publish.
 - **[Warning]** Author eslesar;mgreenegit cannot be recognized in GitHub.
 - **[Warning]** Can&#39;t find output file for bread/toc.json

<a name="dsc%2FmetaConfig4.md"></a>
### [dsc/metaConfig4.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig4.md)
 - **[Warning]** Ignore metadata: ms.locale. They are generated by open publish.
 - **[Warning]** Author eslesar;mgreenegit cannot be recognized in GitHub.
 - **[Warning]** Can&#39;t find output file for bread/toc.json

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C12%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201710122028045818-1754%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FKIP4Sk6oTJv809isqfnqQTkaLgkRNM9vZO87ICMIpM%3D&st=2017-10-12T20%3A31%3A07Z&se=2017-11-12T20%3A36%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: dsc/metaConfig.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-ms.date:  2017-06-12
-author:  eslesar
+ms.date:  2017-10-11
+author:  eslesar;mgreenegit
```
Comment (zjalexander) [inline dsc/metaConfig.md]: This isn't neccessary

File: dsc/metaConfig4.md [hunk 2]
```diff
@@ -1,38 +1,46 @@
 ---
-ms.date:  2017-06-12
-author:  eslesar
+ms.date:  2017-10-12
+author:  eslesar;mgreenegit
```
Comment (zjalexander) [inline dsc/metaConfig4.md]: this isn't neccessary

**Comment by juanpablojofre** [comment]
Closing and re-opening to force new build.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/metaConfig.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig.md) | :warning:Warning | | [Details](#user-content-dsc%2FmetaConfig.md)
[dsc/metaConfig4.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig4.md) | :warning:Warning | | [Details](#user-content-dsc%2FmetaConfig4.md)
[dsc/TOC.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/TOC.md) | :white_check_mark:Succeeded | |

<a name="dsc%2FmetaConfig.md"></a>
### [dsc/metaConfig.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig.md)
 - **[Warning]** Ignore metadata: ms.locale. They are generated by open publish.
 - **[Warning]** Author eslesar;mgreenegit cannot be recognized in GitHub.
 - **[Warning]** Can&#39;t find output file for bread/toc.json

<a name="dsc%2FmetaConfig4.md"></a>
### [dsc/metaConfig4.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-1/dsc/metaConfig4.md)
 - **[Warning]** Ignore metadata: ms.locale. They are generated by open publish.
 - **[Warning]** Author eslesar;mgreenegit cannot be recognized in GitHub.
 - **[Warning]** Can&#39;t find output file for bread/toc.json

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C13%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201710131711525371-1754%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=badUFEkkG0yu496Ncor4Trbl%2BZ5s427sef0WNuOH1bk%3D&st=2017-10-13T17%3A38%3A58Z&se=2017-11-13T17%3A43%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**